Tests/scripts/create_artifacts_graph/create_artifacts.py

```diff
@@ -51,6 +60,9 @@ def main():
     parser.add_argument("-mp", "--marketplace", type=MarketplaceVersions, help="marketplace version", default="xsoar")
     parser.add_argument("-ao", "--artifacts-output", help="Artifacts output directory", required=True)
     parser.add_argument("-do", "--dependencies-output", help="Dependencies output directory", required=True)
+    parser.add_argument(
+        "-bu", "--bucket-upload", help="Upload to bucket", type=lambda x: str2bool(x or False), default=False
```

Comment:
```suggestion
        "-bu", "--bucket-upload", help="Upload to bucket", type=str2bool, default=False
```
That should work too

Reply:
The bucket upload value is empty string, so it fails.